src/main/scala/scala/tools/refactoring/sourcegen/TreeChangesDiscoverer.scala

```diff
@@ -156,9 +156,16 @@ trait TreeChangesDiscoverer {
           List((changed, orig.pos, Set(changed) ++ searchChildrenForChanges(changed)))
         }
 
+        def drillDownChildrenAndCollectChangesWithPosition(parent: Tree): List[(Tree, Position, Set[Tree])] =
```

Comment:
Hmm, OK, I understand... I also don't know if this is the best place to fix it, but maybe you can add what you just explained to me as a comment?